requirements/edx-sandbox/py35.in

```diff
@@ -23,7 +23,7 @@ openedx-calc
 pyparsing==2.2.0                    # Python Parsing module
 random2                             # Implementation of random module that works identically under Python 2 and 3
 scipy==1.2.1                        # Math, science, and engineering library
-sympy==1.4                          # Symbolic math library
+sympy==1.6.2                         # Symbolic math library
```

Comment:
@awais786 This is a risky upgrade; the reason these are pinned in the input requirements file is that there is code provided by instructors that doesn't have any CI and uses these packages.  sympy, numpy, and scipy make backwards incompatible changes in almost every release, and it's hard to figure out what might break code in a course somewhere without asking all course teams to manually test their code in a sandbox with the new version (which we'll be doing soon for the Python 3.5 -> 3.8 switch).  Since this is already in prod I'm inclined to wait and see if it actually triggers any problems (in which case we'll need to revert), but please keep this in mind for future codejail requirements upgrades.